userdata/testAPI.py

```diff
@@ -0,0 +1,18 @@
+'''
+Created on Mar 16, 2015
+
+@author: raju332
+'''
+import VeloAPI

```

Comment:
Its the test that Bibi gave us for the VeloAPI.py. The whole userdata file should be in gitignore.